pkg/server/collector/trivy.go

```diff
@@ -40,7 +40,7 @@ func NewTrivyCollector(
 			Namespace: namespace,
 			Name:      "vulnerabilities",
 			Help:      "Vulnerabilities detected by trivy",
-		}, []string{"image", "vulnerabilityId", "pkgName", "installedVersion", "severity"}),
+		}, []string{"image", "vulnerabilityId", "pkgName", "installedVersion", "severity", "FixedVersion"}),
```

Comment:
Thank you for your contribution!

The label should be lower camel case such as `fixedVerion` :)

Reply:
Thanks for pointing this out. I guess that was blind copy && paste. Fixed.